SOURCES/src/kernel/MAKETERM.MAK

```diff
@@ -29,6 +29,8 @@
 #
 # cbk 06/22/92  added v4term.sys for DEC VT420
 #
+# jts 01/20/19	edits to use missing files out of ..\mos5src
+#

```

Comment:
I don't think we need to add to this history, that's what commit messages are for in git. For example
`git log SOURCES/src/kernel/MAKETERM.MAK` will tell you the change history for just that file.
`git blame SOURCES/src/kernel/MAKETERM.MAK` will show you the file with the commit id that changed each line.

Reply:
Maybe some people don't use git. I will not install git on my DOS PC where I build the source. All I need is a ZIP file once per week. If I make any patches, I can email them, and the git gurus can have all the git fun to themselves. No offense intended, but I'm old, and learning git is a waste of my time.

Reply:
You don't need to install anything, you can do it all from github and it's probably easier than replying to this thread. Click on the file in the tree you're interested in e.g.
https://github.com/roelandjansen/pcmos386v501/blob/master/SOURCES/src/MAKEMOS.BAT
Look towards the top right, there are three buttons, `raw`, `blame` and `history`, they'll do the same as I posted above. So you can see what was done to the file, when, by whom, and the commit message explaining why.

Reply:
Never mind, I retracted it.  I created a branch if you are interested @src153.

Reply:
@andrewbird thanks for trying to be helpful, but web interfaces are another thing I don't like.
@the-grue A branch? Something else to learn? I'lll let you git gurus fight it out.

No disrespect to Roeland, but just because Roeland did it a certain way, does not make it the best way. Project owners deserve no special consideration.

Reply:
I don't think it is a matter of how Roeland did it.  I believe he just dumped a backup tape/disk, cleaned it up a bit to remove potential IP issues, and uploaded it.

As for the branch, if you go to my fork and click on Branch and change it from Master to makefiles you can navigate to the two files I changed in kernel (MAKEUTIL.MAK and MAKETERM.MAK).  Not big changes, just enough to get everything building under kernel.

As for git guru, I am far from it.  Which should be obvious from my recent uploads!  Haha!